### PR TITLE
VIC submissions

### PR DESCRIPTION
**Pull request by lihanli**
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7854
TODO: MVI historical ICNs

### CODE REVIEW

File: app/controllers/v0/vic/vic_submissions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    class VICSubmissionsController < ApplicationController
+      skip_before_action(:authenticate)
+
+      def create
+        vic_submission = ::VIC::VICSubmission.new(
+          params.require(:vic_submission).permit(:form)
+        )
+        vic_submission.user_uuid = current_user.uuid if current_user.present?
+
+        unless vic_submission.save
```
Comment (rlbaker) [inline app/controllers/v0/vic/vic_submissions_controller.rb]: Should this be `save!`?

**Comment by jbalboni** [comment]
@lihanli I get this error when I submit locally through the front end:

`Restforce::UnauthorizedError: INVALID_SESSION_ID: Session expired or invalid`

Is that expected?

**Comment by lihanli** [comment]
@jbalboni yea, the private key is not going to work locally, only the keys on dev and staging work

File: app/controllers/v0/vic/vic_submissions_controller.rb (hunk 1 above)
Comment (lihanli) [reply app/controllers/v0/vic/vic_submissions_controller.rb]: i raise on line 19 after logging to sentry so don't need to raise on the save

**Pull request merged**